src/cairo/Surface.d

```diff
@@ -120,12 +115,7 @@ public class Surface
 	{
 		this.cairo_surface = cairo_surface;
 	}
-	
-	this(Window window, cairo_content_t content, int width,int height)
```

Comment:
Ideally we want to keep this constructor for the copy of cairo that is bundled with the gtk3 binding.

Although the plan is to break up gtkD into smaller libraries, and then it will have to be moved to gdk.